index.src.html

```diff
@@ -778,7 +788,9 @@ <h4 id="fetching">Fetching</h4>
           [=request/policy container=] accordingly, or to directly set
           |request|'s [=request/policy container=] to a non-null value.
 
-      1.  If |request|'s [=request/target IP address space=] is not null, then:
+      1.  If |request|'s [=request/origin=] is a [=potentially trustworthy
+          origin=] and |request|'s [=request/target IP address space=] is not
+          null, then:
```

Comment:
I don't think we should make this change.  The target IP address space can be non-null regardless whether the request's origin is potentially trustworthy.
- When the request's origin is *not* potentially trustworthy, the target IP address space can be added by the `fetch()` API.
- When the request's origin is potentially trustworthy, we might also add the target IP address space when if the initial connection is unexpectedly to a less-public IP address space.

In both cases, we need to check whether the connection's IP address space matches the target IP address space.
```suggestion
      1.  If |request|'s [=request/target IP address space=] is not null, then:
```

Reply:
I don't think that's true. Target address space is only for relax mixed content check. It there's no mixed content to relax, then `targetAddressSpace` is irrelevant and we should simply ignore it.

Reply:
We talked about this offline.  The `targetAddressSpace` can also be set when an non-mixed-content fetch's initial connection fails because its IP address space is less-public than the request policy container's.